dev/Notifications/BadgeNotifications/BadgeNotificationManager.cpp

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "BadgeNotificationManager.h"
+#include "Microsoft.Windows.BadgeNotifications.BadgeNotificationManager.g.cpp"
+#include "externs.h"
+#include <frameworkudk/toastnotifications.h>
```

Comment:
BadgeNotificationManager should only delegate the call to BadgeNotification class.
the framework udk handling should go happen on BadgeNotification class.
BadgeNotificationManager, should only call the methods on BadgeNotification, to the functionality done.

Reply:
BadgeNotification class is just a struct that carries information about the notification. It should not have any functionality. So all the operation happens on manager level and all the information is present in badgenotification class